kubert/src/runtime.rs

```diff
@@ -553,3 +589,14 @@ impl LogSettings {
         self.format.try_init(self.filter)
     }
 }
+
+// === impl RuntimeMetrics ===
+
+impl RuntimeMetrics {
+    /// Creates a new set of metrics and registers them.
+    #[cfg(feature = "prometheus-client")]
+    pub fn new(registry: &mut Registry) -> Self {
```

Comment:
```suggestion
    pub fn register(registry: &mut Registry) -> Self {
```

I prefer to use a convention of using `register` for these types, reserving `new` for constructors that aren't side-effecty.